draft-ietf-regext-rdap-geofeed.md

```diff
@@ -32,107 +32,208 @@ email = "tomh@apnic.net"
-extension string in the `rdapConformance` array for the IP Network lookup and
-search responses, as well as in the help response.
+An RDAP server conforming to this specification MUST include the "geofeed1" extension string in the "rdapConformance"
+array for the IP Network lookup and search responses, as well as in the help response.
 
-## Geofeed Member
+## Geofeed Link
 
-The IP Network object class is extended to include a new `geofeed` member, as
-specified below:
+The IP Network object class ([@!RFC9083, section 5.4]) MAY include a link object for the geofeed file URL in its "links"
+array, with the following REQUIRED JSON members:
 
-geofeed — a string specifying the HTTPS URL of the geofeed file for an IP
-          network [@!RFC9092]
+* "value" -- The "value" JSON value is the context URI and set to an IP Network lookup URL ([@!RFC9082, section 3.1.1]).
+* "rel" -- The "rel" JSON value is the link relation type and set to the "geofeed" string. The "geofeed" link relation
+type is new and will be registered in the IANA Link Relations Registry (see the "Link Relations Registry" section).
+* "href" -- The "href" JSON value is the target URI and set to the HTTPS URL of the geofeed file for the IP network in
+the context URI.
 
-Since this specification extends the base definition of the IP Network object
-class, per [@!RFC7480, section 6], the `geofeed` member name MUST be prefixed
-with the `geofeedv1` extension string, followed by an underscore:
-`geofeedv1_geofeed`.
+Per the definition of a web link ([@!RFC8288]), a "geofeed" link object MAY have additional OPTIONAL JSON members.
+Specifically:
 
-The `geofeed` member is OPTIONAL for the `geofeedv1` extension. Furthermore,
-the Redaction by Removal method [@?I-D.ietf-regext-rdap-redacted] SHOULD be
-used when redacting this member.
+* "type" -- The "type" JSON value is the media type for the target URI. See the "Media Type for a Geofeed Link" section
+for acceptable values.
+* "hreflang" -- The "hreflang" JSON value is an attribute for the target URI and could be used to indicate the languages
+the geofeed data is available in.
+
+There MAY be zero or more "geofeed" link objects in the "links" array of an IP Network object. In other words, the
+"geofeed" link objects are OPTIONAL.
+
+## Media Type for a Geofeed Link
+
+[@?I-D.ymbk-opsawg-9092-update] requires a geofeed file to be a UTF-8 CSV file, with a series of "#" comments at the end
+for the optional RPKI signature. The "text/csv" media type ([@!RFC7111, section 5.1]) seems like a good candidate to
+represent a geofeed file, but it does not support the "#" comments needed to include the RPKI signature.
+
+To enable including "#" comments for an RPKI signature, a new media type "application/geofeed+csv" will be registered in
+the IANA Media Types Registry (see the "Media Types Registry" section).
+
+It is RECOMMENDED that the "type" JSON value in a "geofeed" link object be set to "application/geofeed+csv" media type.
```

Comment:
Do we want to say something about "application/geofeed+..." with other sub-types in the future? Part of me says we should align tightly with the RFC 9092 bis format for now, and let this evolve organically in the future.

Reply:
Ok. Good observation on that additional registry.